boards/arm/raytac_mdbt50q_db_40_nrf52840/doc/index.rst

```diff
@@ -0,0 +1,212 @@
+.. _raytac_mdbt50q_db_40_nrf52840:
+
+MDBT50Q-DB-40
```

Comment:
(optional) Might want to put Raytac at the front here so it appears with that in the documentation list